src/themes/elife/custom-properties.css

```diff
@@ -0,0 +1,3 @@
+:root {
```

Comment:
Use `:--root` to be consistent with the rest of the codebase. Might need to change it later to widen the scope to this, but for now let's start with consistency.